docs/faq.rst

```diff
@@ -0,0 +1,190 @@
+Frequently Asked Questions
+==========================
+
+Those questions have been asked in `GitHub issues`__ for the project by
+several people. If you have more questions, feel free to open a new issue,
+and your question and its answer might make it to this FAQ.
+
+__ https://github.com/citusdata/pg_auto_failover/issues_
+
+The secondary is blocked in the CATCHING_UP state, what should I do?
+--------------------------------------------------------------------
+
+In the pg_auto_failover design, the following two things are needed for the
+monitor to be able to orchestrate nodes integration completely:
+
+ 1. Health Checks must be successful
+
+    The monitor runs periodic health checks with all the nodes registered
+    in the system. Those *health checks* are Postgres connections from the
+    monitor to the registered Postgres nodes, and use the ``hostname`` and
+    ``port`` as registered.
+
+    The ``pg_autoctl show state`` commands column *Reachable* contains
+    "yes" when the monitor could connect to a specific node, "no" when this
+    connection failed, and "unknown" when no connection has been attempted
+    yet, since the last startup time of the monitor.
+
+    The *Reachable* column from ``pg_autoctl show state`` command output
+    must show a "yes" entry before a new standby node can be orchestrated
+    up to the "secondary" goal state.
+
+ 2. pg_autoctl service must be running
+
+    The pg_auto_failover monitor works by assigning goal states to
+    individual Postgres nodes. The monitor will not assign a new goal state
+    until the current one has been reached.
+
+    To implement a transition from the current state to the goal state
+    assigned by the monitor, the pg_autoctl service must be running on
+    every node.
+
+When your new standby node stays in the "catchingup" state for a long time,
+please check that the node is reachable from the monitor given its
+``hostname`` and ``port`` known on the monitor, and check that the
+``pg_autoctl run`` command is running for this node.
+
+When things are not obvious, the next step is to go read the logs. Both the
+output of the ``pg_autoctl`` command and the Postgres logs are relevant. See
+the :ref:`logs` question for details.
+
+.. _logs:
+
+Should I read the logs? Where are the logs?
+-------------------------------------------
```

Comment:
Out of  curiosity again, is there a way to increase the pg_autoctl log level while it is running?

Reply:
Not at the moment. We could use SIGUSR1 and SIGUSR2 to mean logmore/logless and change the debug level dynamically with that, or maybe have the verbosity in the configuration file and allow to change it on reload, but we have nothing of the sorts at the moment. You could open an issue to make it happen later ;-)